app/(root)/om-oss/page.tsx

```diff
@@ -1,9 +1,21 @@
-import React from "react"
+import InfoPageContent from "@/components/omOss/InfoPageContent";
+import { CircularProgressIndicator } from "@/components/suspense"
+import { Suspense } from "react"
+import { getAllInfoPages } from "@/sanity/queries/omOss"
 
-export default function OmOssPage() {
+export const dynamic = "force-dynamic" //TODO Må den være her
```

Comment:
Trengs vel egentlig ikke her. Det er bare for å slå av caching fra next. Men det er ikke så farlig om innholdet i denne siden blir cachet